src/test/java/racinggame/domain/CarTest.java

```diff
@@ -0,0 +1,44 @@
+package racinggame.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import racinggame.domain.Car;
+import racinggame.domain.MoveCondition;
+import racinggame.domain.Position;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("도메인 객체 Car 테스트")
+public class CarTest {
+
+    private Car car;
+
+    @BeforeEach
+    void initCar() {
+        car = new Car();
+    }
+
+    @Test
+    public void 생성자_초기위치_0() {
+        assertThat(car.position()).isEqualTo(new Position(0));
```

Comment:
기본 생성자로 `Position` 인스턴스를 생성할 때 초기 위치 값을 `0`으로 부여하도록 해보면 어떨까요?
```suggestion
        assertThat(car.position()).isEqualTo(new Position());
```